DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/ViewDelegateController.cs

```diff
@@ -116,7 +117,8 @@ DelegateAccessRoute accessedVia
                 customisationId,
                 course!.CourseName,
                 false,
-                accessedVia
+                accessedVia,
+                returnPage

```

Comment:
We are also going to want to maintain the return page value when a validation error occurs during the post. Currently, I don't think it will be retained, so the cancel link will return the user back to the first page if they hit the submit button without checking the checkbox. 

Can't remember if we just add the return page as a query parameter to the post, or have a hidden input somewhere to keep it in the model. Have a look at another example elsewhere and do what that does. E.g. All Delegates --> Set password.

Reply:
Good spot! 👍